tests/rkt_image_rm_test.go

```diff
@@ -265,12 +357,15 @@ func getImageId(ctx *testutils.RktRunCtx, name string) (string, error) {
 	return imageID, nil
 }
 
-func removeImage(ctx *testutils.RktRunCtx, image string, shouldWork bool) error {
-	expect := fmt.Sprintf(rmImageReferenced, image)
+func removeImage(ctx *testutils.RktRunCtx, images []string, shouldWork bool) error {
+	cmd := fmt.Sprintf("%s image rm", ctx.Cmd())
+	for _, image := range images {
+		cmd += fmt.Sprintf(" %s", image)
+	}
+	expect := fmt.Sprintf(rmImageReferenced, images[0])
```

Comment:
I'm not sure how this should be handled, i.e. multiple images and `shouldWork` set to `false`. On the other hand, there is no test case where `shouldWork` actually is `false`.